content/en/docs/collector/architecture.md

```diff
@@ -0,0 +1,315 @@
+    PN --> FO((fan out))
+    FO --> E1[[Exporter 1]]
+    FO --> E2[[Exporter 2]]
+    FO ~~~ EM[[...]]
+    FO --> EN[[Exporter N]]
+
+    %% The stroke color matches the website header.
+    classDef default fill:#e3e8fc,stroke:#4f62ad
+```
+
+Pipelines can have one or more receivers. Data from all receivers is pushed to
+the first processor, which processes the data and then pushes it to the next
+processor (or a processor may drop the data if it is a “sampling” processor, for
+example), and so on until the last processor in the pipeline pushes the data to
+the exporters. Each exporter gets a copy of each data element. The last
+processor uses a `fanoutconsumer` to fan out the data to multiple exporters.
+
+The pipeline is constructed during Collector startup based on pipeline
+definition in the configuration.
+
+A pipeline configuration typically looks like this:
+
+```yaml
+service:
+  pipelines: # section that can contain multiple subsections, one per pipeline
+    traces: # type of the pipeline
+      receivers: [otlp, zipkin]
+      processors: [memory_limiter, batch]
+      exporters: [otlp, zipkin]
+```
+
+The above example defines a pipeline for the traces type of telemetry data,
+with three receivers, two processors, and three exporters.
+
+For details of config file format see
+[this document](https://docs.google.com/document/d/1NeheFG7DmcUYo_h2vLtNRlia9x5wOJMlV4QKEK05FhQ/edit#).
+
+### Receivers
+
+Receivers typically listen on a network port and receive telemetry data. Usually
+one receiver is configured to send received data to one pipeline. However, it is
+also possible to configure the same receiver to send the same received data to
+multiple pipelines. This can be done by listing the same receiver in the
+“receivers” key of several pipelines:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+
+service:
+  pipelines:
+    traces: # a pipeline of “traces” type
+      receivers: [otlp]
+      processors: [memory_limiter, batch]
+      exporters: [jaeger]
+    traces/2: # another pipeline of “traces” type
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [otlp]
+```
+
+In the above example, `otlp` receiver will send the same data to pipeline
+`traces` and to pipeline `traces/2`. (Note: The configuration uses composite key
+names in the form of `type[/name]` as defined in
+[this document](https://docs.google.com/document/d/1NeheFG7DmcUYo_h2vLtNRlia9x5wOJMlV4QKEK05FhQ/edit#)).
+
+When the Collector loads this config, the result will look like this diagram
+(part of processors and exporters are omitted for brevity):
+
+<!--TODO: Add Receivers image via Mermaid.-->
+
+
+{{% alert title="Important" color="warning" %}}
+ 
+When the same receiver is referenced in more than one pipeline, the
+Collector creates only one receiver instance at runtime that sends the data to
+a fan out consumer. The fan out consumer in turn sends the data to the first
```

Comment:
I'm not the original author, so I wasn't sure what "fan out" means either. I read through the processor README and found it [mentioned there](https://github.com/open-telemetry/opentelemetry-collector/blob/4ae8d2bcd5ae2e72280fb6af7e9402b7b6e7c6cb/processor/README.md?plain=1#L45) as well. From a [quick web search](https://www.google.com/search?q=fan-out+connector+software&sca_esv=59bcec2abd4eef4d&ei=NQzdZcfZEr22wN4Px-qMwAs&ved=0ahUKEwiHyKHcgsqEAxU9G9AFHUc1A7gQ4dUDCBA&uact=5&oq=fan-out+connector+software&gs_lp=Egxnd3Mtd2l6LXNlcnAiGmZhbi1vdXQgY29ubmVjdG9yIHNvZnR3YXJlMgUQIRigATIFECEYoAFIsxRQzwZYvhJwAXgBkAEAmAGdAaAB6QmqAQMwLjm4AQPIAQD4AQGYAgqgAo8KwgIKEAAYRxjWBBiwA8ICBhAAGBYYHsICCBAAGBYYHhgPwgILEAAYgAQYigUYhgOYAwCIBgGQBgiSBwMxLjk&sclient=gws-wiz-serp), "fan-out" seems like a recognized term for a software concept. 

I will add the hyphen. I can also write a parenthetical definition to include in the documentation, but it would be good to have an SME check it over. I'll tag one of the Collector folks once I've pushed the changes.

Reply:
Let's leave that wording as is. I agree with @theletterf that not everyone would know what "fan out" means, but we are taking the original document over, so I don't think we should change this right now